lib/helper-functions.js

```diff
@@ -18,5 +21,36 @@ const componentNameToMacroName = componentName => {
 
   return `govuk${macroName}`
 }
-
 exports.componentNameToMacroName = componentNameToMacroName
+
+// Read components from various sources
+exports.SrcComponentList = fs.readdirSync(configPaths.src + 'components/')
+exports.DistComponentList = fs.readdirSync(configPaths.dist + 'components/')
+exports.PackagesComponentList = fs.readdirSync(configPaths.packages)
+
+// Compare component files from src with dist
+const compareComponentDist = (componentName) => {
+  let distComponentFiles = fs.readdirSync(configPaths.dist + 'components/' + componentName)
+  // We don't want to include the yaml file in dist
```

Comment:
or the readme template?

Reply:
Yes, if we don't want `Readme.md`s in `dist` I can exclude them. I was looking at what the tasks are supposed to be doing.

Reply:
Sorry, the readme _template_ (`index.njk`) – which is not mentioned in this comment.